src/v/serde/serde.h

```diff
@@ -347,6 +351,10 @@ void write(iobuf& out, T t) {
     } else if constexpr (std::is_same_v<Type, bytes>) {
         write<serde_size_t>(out, t.size());
         out.append(t.data(), t.size());
+    } else if constexpr (std::is_same_v<Type, uuid_t>) {
+        for (const uint8_t byte : t.uuid) {
+            write(out, byte);
+        }
```

Comment:
should this be: `out.append(t.uuid.data, uuid_t::length)`?

Reply:
Done